notebooks/OFC/truncation_thresholds.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>Can you just add a little intro explaining this notebook and what it does?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>